### PR TITLE
unfreeze nightly ci after syntex-syntax update

### PR DESCRIPTION
**Pull request by Goirad**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ addons:
 rust:
    - stable
    - beta
+   - nightly
    - nightly-2019-07-01
    - nightly-2019-07-11
```
Comment (jethrogb) [inline .travis.yml]: Shouldn't one of these have been removed?

Comment (Goirad) [reply .travis.yml]: The intention is to update 07-11 to whatever the next nightly we end up using for our toolchain is

**Pull request merged**